contributors/design-proposals/propagation.md

```diff
@@ -109,6 +125,12 @@ Opinion against this:
 
 1. (From @jonboulle) May cause cross-runtime compatibility issue.
 
+1. It's not possible to validate a pod + mount propagation. Mount propagation
```

Comment:
isn't local volumes using it? pv generated with hostpath and injected attribute to pin it to a particular host?